nautobot/core/admin.py

```diff
@@ -11,3 +12,8 @@
 
 # Unregister the unused stock Tag model provided by django-taggit
 admin_site.unregister(Tag)
+
+# Unregister SocialAuth from DJango admin menu
```

Comment:
```suggestion
# Unregister SocialAuth from Django admin menu
```

Reply:
> Why remove these? I can see them potentially being useful in environments which use SSO

@nniehoff currently we have no usecases for these models so think it would be cleaner to disable them. Do you know of any use cases or have you used these models with Nautobot?